src/main/java/de/tum/in/www1/artemis/web/rest/PlagiarismResource.java

```diff
@@ -35,9 +35,9 @@ public PlagiarismResource(PlagiarismComparisonRepository plagiarismComparisonRep
      * parameters are invalid
      */
     @PutMapping("/plagiarism-comparisons/{comparisonId}/status")
-    @PreAuthorize("hasRole('INSTRUCTOR')")
+    @PreAuthorize("hasRole('EDITOR')")
     public ResponseEntity<Void> updatePlagiarismComparisonStatus(@PathVariable long comparisonId, @RequestBody PlagiarismComparisonStatusDTO statusDTO) {
-        // TODO: check that the instructor has access to the corresponding course (add the exerciseId to the URL)
+        // TODO: check that the editor has access to the corresponding course (add the exerciseId to the URL)
```

Comment:
Maybe you could a the check here. If I remember correctly it shouldn't be hard in this context

Reply:
I guess it is too late for that now since the PR is already merged. 
I have had the feeling that I might overlook something that has to be done in this resource, since the way I would have approached it wouldn't have been much more effort than writing the comment. That's why I just updated it - but you're right, next time I should have a closer look at it.